recipes-domu/domu-image-android/files/0005-fetch2-repo-Add-group-parameter.patch

```diff
@@ -0,0 +1,28 @@
+From f9b84d2ae9af8e4aa23f3d29b2c5675b80a38c1f Mon Sep 17 00:00:00 2001
+From: xen-troops-buildbot <not.real.email@epam.com>
+Date: Tue, 11 Dec 2018 15:10:42 +0000
+Subject: [PATCH] Add group parameter
+
+---
+ bitbake/lib/bb/fetch2/repo.py | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/bitbake/lib/bb/fetch2/repo.py b/bitbake/lib/bb/fetch2/repo.py
+index ff4d548..ba6df27 100644
+--- a/bitbake/lib/bb/fetch2/repo.py
++++ b/bitbake/lib/bb/fetch2/repo.py
+@@ -77,7 +77,11 @@ class Repo(FetchMethod):
+
+         bb.utils.mkdirhier(ud.repodir)
+         bb.fetch2.check_network_access(d, "repo init -m %s -b %s -u %s://%s%s%s" % (ud.manifest, ud.branch, ud.proto, username, ud.host, ud.path), ud.url)
+-        runfetchcmd("repo init -m %s -b %s -u %s://%s%s%s" % (ud.manifest, ud.branch, ud.proto, username, ud.host, ud.path), d, workdir=ud.repodir)
++        prebuilds = d.getVar("TARGET_PREBUILT_KERNEL") or ""
```

Comment:
This is not fetcher's job to know if you are building from sources or not.
IMO, the right approach would be to add "group" parameter, so fetcher can use it, e.g. the same way as we set "branch", "host", "proto" parameters - see ud.branch etc.